.pre-commit-config.yaml

```
@@ -29,4 +29,8 @@ repos:
         args:
           - --profile=black
           - --filter-files
-          - --line-length=79
\ No newline at end of file
+          - --line-length=79
+- repo: https://github.com/codespell-project/codespell
+  rev: v2.2.2
+  hooks:
+  - id: codespell
```

```suggestion
  - repo: https://github.com/codespell-project/codespell
    rev: v2.2.2
    hooks:
    - id: codespell
```
I think this block needs to be indented similar to the other hooks, otherwise pre-commit has trouble parsing it when run!

---

oh, thanks and sorry about that! I must fix my helper https://github.com/yarikoptic/improveit/blob/HEAD/codespellit somehow -- different projects might have different ideas on how to indent yaml files. You like 2 spaces here whenever e.g. https://github.com/pydantic/pydantic/blob/main/.pre-commit-config.yaml used none and thus I had it like this here.

I thought now I could use round trip feature of ruamel.yaml but it seems it also kills your indentation (the command I tried was `python3 -c 'from ruamel.yaml import YAML; from pathlib import Path; path=Path(".pre-commit-config.yaml"); yaml = YAML(typ="rt"); outdata = yaml.load(path); yaml.dump(outdata, path)'`).  so ok, for now just manually and will need to keep attention in the future! ;)